src/components/NoticeText/CampaignRegistration10DlcNoticeText.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const CampaignRegistration10DlcNoticeText: React.FC = () => (
+  <>
+    <p>
+      <b>
+        You have started but not yet completed your 10DLC registration form.
+      </b>
+      <br /> <br />
```

Comment:
I would prefer separating these out as paragraphs rather than using the line break tag.